RubeesEat/appsettings.json

```diff
@@ -10,5 +10,10 @@
     "DefaultConnectionString": "Server=localhost;Database=RubeesEatDB;UID=root;PWD=;",
     "TestConnectionString": "Server=localhost;Database=RubeesEatTestDB;UID=root;PWD=;"
   },
+  "OpenId": {
+    "ClientId": "%OpenId__ClientId%",
+    "ClientSecret": "%OpenId__ClientSecret%",
+    "Authority": "%OpenId__Authority%"

```

Comment:
I"m assuming because we had placeholders/default values for the connectionsstrings (which are anyhow replaced by the env variables).

Should we just remove these alltogether? It works without the OpenID part in the appsettings, since it is anyhow gotten from the env variables.